views/items/show.handlebars

```diff
@@ -22,15 +22,21 @@
 
 <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
   <div class="carousel-inner">
-    <div class="carousel-item active">
-      <img class="carousel_image" src="https://scontent.fewr1-6.fna.fbcdn.net/v/t45.5328-4/278302928_4890743744306143_6901770805173009719_n.jpg?stp=dst-jpg_s960x960&_nc_cat=109&ccb=1-5&_nc_sid=c48759&_nc_ohc=LxuZ5gKGzpsAX_r-lSM&_nc_ht=scontent.fewr1-6.fna&oh=00_AT9-X1yAtK2_zRQei5fi3m7gjntERmM0MwhSJm8ElMObWQ&oe=6275D679" alt="First slide">
-    </div>
-    <div class="carousel-item">
-      <img class="carousel_image" src="https://scontent.fewr1-5.fna.fbcdn.net/v/t45.5328-4/278674873_5177796068937602_2031184812532681317_n.jpg?stp=dst-jpg_s960x960&_nc_cat=106&ccb=1-5&_nc_sid=c48759&_nc_ohc=gcpE1H662VUAX9bY50F&_nc_ht=scontent.fewr1-5.fna&oh=00_AT-S8wibGSpWuefTUBJ6AzVY9OdCxoed9wVlJ2CeWxwRyQ&oe=6276053C" alt="Second slide">
-    </div>
-    <div class="carousel-item">
-      <img class="carousel_image" src="https://scontent.fewr1-5.fna.fbcdn.net/v/t45.5328-4/278650834_5071419116304472_340677323595229689_n.jpg?stp=dst-jpg_s960x960&_nc_cat=105&ccb=1-5&_nc_sid=c48759&_nc_ohc=MEYdao8a76cAX_VnFty&_nc_ht=scontent.fewr1-5.fna&oh=00_AT9Mmg4mpMxHiCe20UUpao3SiQmvmXAHZFKB8yO-llZVfw&oe=627637CE" alt="Third slide">
-    </div>
+    {{#if item.photos}}
+
+      {{#each item.photos}}
+        <div class="carousel-item {{#if @first }}active {{/if}}">
+          <img class="carousel_image" src="{{this.imageUrl}}" alt="{{this.description}}">
+        
+          <div class="carousel-caption d-none d-md-block">
+            <h3>{{this.description}}</h3>
+          </div>
+        </div>
+      {{/each}}
+
+    {{else}}
+      <p>There are no images yet. <a href="/items/{{itemId}}/editPhotos" class="btn btn-primary">Click here</a> to add!</p>
```

Comment:
Only the item owner should see this

Reply:
For a regular user, I think we can something like "Ask the owner in the comments to add photos"